PWGHF/TableProducer/candidateCreatorCascade.cxx

```diff
@@ -112,6 +141,14 @@ struct HfCandidateCreatorCascade {
       const auto& trackV0DaughPos = v0.posTrack_as<MyBigTracks>();
       const auto& trackV0DaughNeg = v0.negTrack_as<MyBigTracks>();
 
+      auto collision = bach.collision();
+
+      // set the magnetic field from CCDB
+      auto bc = collision.bc_as<o2::aod::BCsWithTimestamps>();
+      initCCDB(bc, runNumber, ccdb, isRun2 ? ccdbPathGrp : ccdbPathGrpMag, lut, isRun2);
+      bz = o2::base::Propagator::Instance()->getNominalBz(); // z component
```

Comment:
Why did you remove `if (runNumber != bc.runNumber()) {  ... }` (see https://github.com/AliceO2Group/O2Physics/blob/8505ec1/PWGHF/TableProducer/candidateCreator2Prong.cxx#L113-L125). 

In addition, I would also keep the comment that is made there on converted data.

Reply:
Hi @lvermunt , thanks for your comment. I changed it (I took the chance to also update some histograms again).
Btw, I saw the same also in candidateCreatorOmegac https://github.com/AliceO2Group/O2Physics/blob/master/PWGHF/TableProducer/candidateCreatorOmegac.cxx#L107-L110
Maybe one should then update it there too?

Reply:
Hi @DanielSamitz, in the `candidateCreatorOmegac.cxx` it shouldn't be an issue (in my understanding, but please correct me if I'm wrong) due to the iterator (`void process(SelectedCollisions::iterator const& collision, ...`), so there the if-statement would have no effect.

Reply:
Ah, I see. That's what got me confused.